crucible-mir/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # 0.3 -- 2024-08-30
 
+* The calling sequence of ```translateMIR``` has changed: the first argument,
+  which should always have been passed as ```mempty```, has been removed.
+  This will probably require adjustments in downstream callers.
```

Comment:
More than just "probably": it _will_ require adjustment in downstream callers!

Reply:
Well, yes, I think I meant to write "downstream code" that might or might not actually call it (although I suppose it would be pretty odd if something used this and didn't call that...)